rollup-interface/src/state_machine/zk/mod.rs

```diff
@@ -37,6 +40,13 @@ pub trait Zkvm {
         serialized_proof: &'a [u8],
         code_commitment: &Self::CodeCommitment,
     ) -> Result<&'a [u8], Self::Error>;
+
+    /// Same as the function right above, except that instead of returning the output as a serialized array,
```

Comment:
Nit: I guess saying "Same as regular [`verify`] method, except.." so if something is added in between, documents will still make sense.